kitsune/wiki/utils.py

```diff
@@ -119,11 +126,22 @@ def get_featured_articles(product=None, locale=settings.WIKI_DEFAULT_LANGUAGE):
         .exclude(document__is_template=True)
         .order_by("-visits")
         .select_related("document")
+        .annotate(
+            topic_ids=StringAgg(
```

Comment:
This section could benefit from a comment

Reply:
Done.